docs/2.8.0/docs/tools/assistant-build.rst

```diff
@@ -15,21 +15,502 @@ To compile your Daml source code into a Daml archive (a ``.dar`` file), run::
 
 You can control the build by changing your project's ``daml.yaml``:
 
-``sdk-version``
-  The SDK version to use for building the project.
-
 ``name``
   The name of the project.
 
+``version``
+  The project version.
+
 ``source``
   The path to the source code.
 
-The generated ``.dar`` file is created in ``.daml/dist/${name}.dar`` by default. To override the default location, pass the ``-o`` argument to ``daml build``::
+By default, the generated ``.dar`` file is created in ``.daml/dist/${name}-${version}.dar``. To override the default location, pass the ``-o`` argument to ``daml build``::
 
   daml build -o path/to/darfile.dar
 
 
 Build multiple packages
 ***********************
 
-*New multi-package build content goes here!*
+``daml build`` supports multi-package builds, an optional feature for building
+and managing multiple interdependent packages simultaneously. The multi-package
+build feature consists of a configuration file and a set of flags.
+
+You can use multi-package builds to:
+
+-  Configure ``daml build`` to automatically rebuild DARs used in data
+   dependencies, when the source code corresponding to those DARs
+   changes
+-  Build all of the packages in a project simultaneously, using ``daml build --all``
+-  Clean all build artifacts in a project, using ``daml clean --all``
+
+Overview
+========
+
+To configure which packages to build automatically, add the configuration file ``multi-package.yaml`` at the root of your project:
+
+``multi-package.yaml``
+
+.. code:: yaml
+
+   packages:
+   - <path to first package>
+   - <path to second package>
+
+You can then run ``daml build --all`` from the root of your
+project to build all the packages listed in the ``multi-package.yaml`` file. Or you can run ``daml build`` within a package to build that package and its dependencies.
+
+The multi-package feature of ``daml build`` is optional. To use multi-package, 
+create a ``multi-package.yaml`` file that reflects your project structure. 
```

Comment:
The wording here feels a bit to similar to the above "add the configuration file ``multi-package.yaml`` at the root of your project"
Perhaps wording in the negative form of "If you do not wish to use multi-build, omit the multi-package.yaml file from your project structure" or something like this.

Reply:
Thanks for that suggestion. Updated and did some other sprucing